samples/net/http_server/boards/nrf7002dk_nrf5340_cpuapp_ns.conf

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# TF-M
+CONFIG_TFM_PROFILE_TYPE_MEDIUM=y
+CONFIG_PM_PARTITION_SIZE_TFM_SRAM=0x18000
+
+# Optimize Wi-Fi stack to save some memory
+CONFIG_HEAP_MEM_POOL_SIZE=81920
+CONFIG_NRF700X_RX_NUM_BUFS=16
+CONFIG_NRF700X_MAX_TX_AGGREGATION=4
+
+# Newlib
+CONFIG_NEWLIB_LIBC=y
+
+# Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+CONFIG_WPA_SUPP=y
+CONFIG_NET_L2_ETHERNET=y
+
+# IP address config
+CONFIG_NET_CONFIG_INIT_TIMEOUT=0
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
```

Comment:
Given that DHCP is used, maybe there's no point setting static address? That'd make the conn manager to notify interface readiness before we have a "valid" (DHCP assigned) address.

Reply:
It shouldn't be necessary, I have removed it